libs/langchain-scripts/src/build_v2.ts

```diff
@@ -27,6 +28,50 @@ async function asyncSpawn(command: string, args: string[]) {
   });
 }
 
+const deleteFolderRecursive = async function (inputPath: string) {
```

Comment:
Deleting directories on windows does not act the same as mac. Need this recursive func to manage deleting directories w/ contents.